diff -urNad pcsc-tools-pcsc-tools-1.4.25-orig/Makefile pcsc-tools-pcsc-tools-1.4.25/Makefile
--- pcsc-tools-pcsc-tools-1.4.25-orig/Makefile	2015-11-08 19:13:54.000000000 +0200
+++ pcsc-tools-pcsc-tools-1.4.25/Makefile	2017-01-29 14:03:47.103951243 +0200
@@ -6,7 +6,7 @@
 DESTDIR ?= /usr/local
 
 VERSION := $(shell pwd | sed s/.*tools-//)
-CFLAGS := $(CFLAGS) -DVERSION=\"$(VERSION)\" $(PCSC_CFLAGS)
+CFLAGS := -Wall $(SLKCFLAGS) -DVERSION=\"$(VERSION)\" $(PCSC_CFLAGS)
 LDLIBS := $(PCSC_LDLIBS)
 # On xBSD systems use
 #LDLIBS = -lc_r $(PCSC_LDLIBS)
@@ -31,8 +31,8 @@
 	install -d $(DESTDIR)/share/pcsc
 	install -m 644 smartcard_list.txt $(DESTDIR)/share/pcsc
 
-	install -d $(DESTDIR)/share/man/man1/
-	install -m 644 $(MAN) $(DESTDIR)/share/man/man1/
+	install -d $(DESTDIR)/man/man1/
+	install -m 644 $(MAN) $(DESTDIR)/man/man1/
 
 clean:
 	rm -f pcsc_scan.o $(BIN) $(MAN)
